### PR TITLE
Adds the 2 new components to the techweb

### PR DESCRIPTION
**Pull request by ItsTisme**
## About The Pull Request

Adds the 2 new components I made in my prior PR to the basic circuit research

also fix an issue with associative pick list

## Why It's Good For The Game

 Being able to print them would be usefull

Reason I didn't notice was properly since I ran on debug station and it never crossed my mind that they where research thingy

## Changelog
:cl:
fix: You can print the new list pick and associative list pick components
fix: Associative list pick works as intended now
/:cl:

### CODE REVIEW

**Review by Time-Green — APPROVED** [review]
@Watermelon914

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -369,6 +369,8 @@
 		"integrated_circuit",
 		"module_duplicator",
 		"usb_cable",
+		"comp_list_pick",
+		"assoc_comp_list_pick"
```
Comment (Mothblocks) [inline code/modules/research/techweb/all_nodes.dm:373]: ABC order

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/modules/research/techweb/all_nodes.dm (hunk 1 above)
Comment (Autisem) [reply code/modules/research/techweb/all_nodes.dm]: also renamed the 2nd var to be inline with the other comps

**Comment by Autisem** [comment]
not sure why checks failing

**Review by Watermelon914 — APPROVED** [review]

**Pull request merged**